cv/qc-cv.obo

```diff
@@ -539,14 +564,1316 @@ is_a: QC:4000009 ! ID based
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3B" NARROW []
+
+[Term]
+id: QC:4000176
+name: Charged peptides ratio 4+ over 2+
+def: "Ratio of 4+ peptide count over 2+ peptide count in identified spectra" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3C" NARROW []
+
+[Term]
+id: QC:4000177
+name: Mean charge in identified spectra
+def: "Mean charge in identified spectra" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000178
+name: Median charge in identified spectra
+def: "Median charge in identified spectra" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000179
+name: Charged spectra ratio +1 over +2
+def: "Ratio of 1+ spectra count over 2+ spectra count in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3A"  RELATED []
+
+[Term]
+id: QC:4000180
+name: Charged spectra ratio +3 over +2
+def: "Ratio of 3+ peptide count over 2+ peptide count in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3B"  RELATED []
+
+[Term]
+id: QC:4000181
+name: Charged spectra ratio +4 over +2
+def: "Ratio of 4+ peptide count over 2+ peptide count in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3C"  RELATED []
+
+[Term]
+id: QC:4000182
+name: Mean precursor charge in all MS2
+def: "Mean precursor charge in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000183
+name: Median precursor charge in all MS2
+def: "Median precursor charge in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000184
+name: Number of different proteins from all PSM
+def: "Number of different protein from all PSM" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000189
+
+[Term]
+id: QC:4000185
+name: Number of identified proteins
+def: "Number of identified proteins at given FDR threshold, first number is the number of proteins (considering sequence only), second number is the FDR threshold applied (negative if no threshold applied)" [PSI:QC]
+comment: The number of identifications are likely to only make local sense within a confined set of identification parameters (identification algorithm, searchspace, FDR method).
+is_a: QC:4000004 ! n-tuple
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000189
+
+[Term]
+id: QC:4000186
+name: Total number of PSM
```

Comment:
of PSMs

Reply:
before or after FDR?

Reply:
this is before, there is a term for 'after given threshold' filtering to each spectra, peptides, proteins

Reply:
Made changes there.